subsys/partition_manager/pm.yml.tfm

```diff
@@ -19,8 +15,13 @@ tfm_primary:
 app_primary:
   span: [app]
 
+#ifndef CONFIG_TFM_BL2_FALSE
```

Comment:
What if the default value is to disable MCUBoot? Not a comment directly to this PR, but I guess this is motivation enough to stop using the "use whatever is default TFM" as an option for different TFM kconfigs.

Reply:
Yes, we have a task for changing that. Until then, IMO we can assume that this doesn't change, i.e. default TFM -> BL2 is enabled.